gdk-pixbuf/src/pixbuf.rs

```diff
@@ -306,6 +306,16 @@ impl Pixbuf {
         }))
     }
 
+    // rust-doc-stripper-ignore-next
+    /// Returns a mutable slice to the pixbuf's pixel data.
+    ///
+    /// This function will cause an implicit copy if the pixbuf was created from read-only data.
+    ///
+    /// Please see the section on [image data](#image-data) for information about how the pixel
```

Comment:
this should probably be a proper link no?

Reply:
I think this should be a proper link? The `Image data` section header has an anchor tag with its href set to `image-data`, and this just links to that (as far I can tell)

Reply:
@bilelmoussaoui ?

Reply:
never mind, didn't notice there was a section that you were referencing